Pipfile

```diff
@@ -0,0 +1,28 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+sphinx-rtd-theme = "==0.5.1"
```

Comment:
I dunno about you, but this feels weird. The point of having the Pipfile/Pipfile.lock split is so you _don't_ have to pin your dependencies in the Pipfile.

Reply:
I'd rather keep them pinned, as I want to control when and which major dependencies get updated. I don't want to accidentally update a major dependency that runs "valid" but secretly changes behavior (this was done with docutils)